x-pack/docs/en/rest-api/security/clear-security-cache.asciidoc

```diff
@@ -0,0 +1,54 @@
+[role="xpack"]
+[[security-api-clear-security-cache]]
+=== Clear security cache API
+++++
+<titleabbrev>Clear security cache</titleabbrev>
+++++
+
+Evicts a subset of all entries from the specified security cache.
+The cache is also automatically cleared on state changes of the security index.
+
+NOTE: The intention for the clear security cache API is to have a single endpoint
+for clearing all security related caches. Currently, it only covers API key
+caches. For clearing user cache, see the <<security-api-clear-cache,Clear user cache API>>.
+For clearing role cache, see the <<security-api-clear-role-cache,Clear roles cache API>>.
+For clearing privileges, see <<security-api-clear-privilege-cache,Clear privileges cache API>>.
+These separate APIs will be migrated over time.
+
```

Comment:
This note here doesn't sound too "professional".
We have APIs and deprecated APIs , but no APIs that should do the job of others but not yet.
We can create an API now that we foresee will be deprecated. There is no rush, people can sit on a version for a long time.
Not a big deal, just my thoughts.

Reply:
I will drop it and document it in the context of API key. See my [other comment](https://github.com/elastic/elasticsearch/pull/59376#discussion_r499199071).